pkgs/top-level/all-packages.nix

```diff
@@ -20807,6 +20807,14 @@ in
 
   openra = callPackage ../games/openra { lua = lua5_1; };
 
+  openra-mods = builtins.listToAttrs (map (attrs: {
+    inherit (attrs.mod) name;
+    value = callPackage ../games/openra-mods ({
+      lua = lua5_1;
+      inherit (gnome3) zenity;
+    } // attrs);
+  }) (import ../games/openra-mods/mods.nix { inherit fetchFromGitHub; }));
```

Comment:
Can this replace `openra`?

Reply:
I considered it, but it will complicate the generic mod package definition, because it is structured around having a mod with an engine. I can try to make the mod optional and adding a bunch of conditionals, but if that becomes to unwieldy, I will keep it as is.

Reply:
Tried it, but it would complicate matters too much, better to keep it packaged separately.